common/metrics/defs.go

```diff
@@ -31,8 +31,9 @@ type (
 
 	// metricDefinition contains the definition for a metric
 	metricDefinition struct {
```

Comment:
I started doing this, but I gave up since we don't want to break any callsites for New*Def, and it'd be weird to me if those returned something other than a metricDefinition object.